pkg/cmd/grafana-cli/commands/reset_password_command.go

```diff
@@ -44,15 +50,35 @@ func resetPasswordCommand(c utils.CommandLine, runner server.Runner) error {
 		logger.Infof("\n")
 		logger.Infof("Admin password changed successfully %s", color.GreenString("✔"))
 	}
-	return err
+
+	if errors.Is(err, ErrAdminCannotBeFound) {
+		logger.Infof("\n")
+		logger.Infof("Admin user cannot be found %s. \n", color.RedString("✘"))
+		admins, err := listAdminUsers(runner.UserService)
+		if err != nil {
+			return fmt.Errorf("failed to list admin users: %w", err)
+		}
+
+		logger.Infof("\n")
+		logger.Infof("Please try to run the command again with specifying the correct user-id (--user-id) from the list below:\n")
```

Comment:
```suggestion
		logger.Infof("Please try to run the command again specifying a user-id (--user-id) from the list below:\n")
```

Reply:
We could potentially pull the list of users first, then use it to locate the admin user automatically, first checking for user with id=1, then looking for username=admin (and maybe forcing the user to specify which one via --user-id if we find both).

Reply:
I would not check for "username=admin", because if the default admin id is not 1 then I think it's better to ask them (by providing a list) to specify the user id explicitly to prevent resetting other administrators password accidentally.